### PR TITLE
SOF-7218: cli example of optical property calculation using simple.x

### PR DESCRIPTION
**Pull request by pranabdas**

### CODE REVIEW

File: espresso/simple.x/output/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Quantum ESPRESSO SIMPLE.x Optical Property Calculation
+
+This folder contains output data files of optical property calculation using
+`simple.x`.
+
+Some of the output files are included in [../output-reference](../output-reference).
```
Comment (timurbazhirov) [inline espresso/simple.x/output/README.md:6]: Let's include a link to the original example?

File: espresso/simple.x/input/09_si_abcoeff_to_eps.in [hunk 2]
```diff
@@ -0,0 +1,10 @@
+100
```
Comment (timurbazhirov) [inline espresso/simple.x/input/09_si_abcoeff_to_eps.in:1]: There's nothing in job.pbs that's using this file currently, why?

File: espresso/simple.x/job.pbs [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+# run the calculations
+mpirun -np 4 pw.x         -i 01_si_scf_k.in      > 01_si_scf_k.out
+mpirun -np 4 head.x       -i 02_si_head.in       > 02_si_head.out
+mpirun -np 4 pw.x         -i 03_si_nscf.in       > 03_si_nscf.out
+mpirun -np 4 pw4gww.x     -i 04_si_pw4gww.in     > 04_si_pw4gww.out
+# `-i filename` may not work with GWW executables, please use std redirection < instead
+mpirun -np 4 gww.x         < 05_si_gww.in        > 05_si_gww.out
+mpirun -np 4 pw.x         -i 06_si_nscf_k.in     > 06_si_nscf_k.out
+mpirun -np 4 simple.x      < 07_si_simple.in     > 07_si_simple.out
+mpirun -np 4 simple_bse.x  < 08_si_simple_bse.in > 08_si_simple_bse.out
```
Comment (timurbazhirov) [inline espresso/simple.x/job.pbs:62]: What's the result of this calculation - is that a plot and/or a number - what are we referencing??

File: espresso/simple.x/input/09_si_abcoeff_to_eps.in (hunk 2 above)
Comment (pranabdas) [reply espresso/simple.x/input/09_si_abcoeff_to_eps.in:1]: This is the input file for postprocessing step, which transforms a, b coefficients to epsilon. Currently, the postprocessing tool is not installed in our clusters.

File: espresso/simple.x/job.pbs (hunk 3 above)
Comment (pranabdas) [reply espresso/simple.x/job.pbs:62]: Currently, we only obtain the a, b coefficients (`ab_coeff.dat`), we need another step to calculate the epsilon.

**Pull request merged**